### PR TITLE
Allowed most mobs to whisper.

### PR DESCRIPTION
**Pull request by ShiftyRail**
[content]

Adapted human whisper to a more abstract level.

:cl:
- rscadd: Most mobs, including pAIs, aliens, and constructs, can now whisper.

### CODE REVIEW

**Comment by DamianX** [comment]
I'm unhappy with the amount of copypaste this would result in if we're going to just duplicate human/whisper() for each mob that we want to be able to whisper.
In #18473 I did something similar to this before abandoning it as usual.

**Comment by ShiftyRail** [comment]
Outisde of MoMMis and borers, what else needed fixing ? Dionae nymphs whispering in common ?

**Comment by DamianX** [comment]
iirc the main issue with that PR was that languages were not respected in general, so you could understand xenos and constructs whispering or similar.

**Comment by ShiftyRail** [comment]
Looks like it works for me
@DamianX Had no problems whatsoever with languages. Tested Xeno, Construct, and Mice. They can understand themselves, have the correct font, and people not understanding the language see foreign.

File: code/modules/mob/living/silicon/mommi/say.dm [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 	return "states, [text]";
 
+/mob/living/silicon/robot/mommi/whisper(message as text)
+	if (emagged)
```
Comment (jknpj) [inline code/modules/mob/living/silicon/mommi/say.dm]: check for KEEPER instead, since it's what makes them unable to talk

Comment (ShiftyRail) [reply code/modules/mob/living/silicon/mommi/say.dm]: Righto.

**Review by Intigracy — CHANGES_REQUESTED** [review]
Check for KEEPER

**Pull request merged**